### ISSUE TITLE
CC Adv Monitor & Chisels & Bits Chisel causes client crash

### ISSUE THREAD

**Issue by Kiefy** [NillerMedDild/Enigmatica2Expert#1179]
**Modpack version**
1.62

**Can the issue be reproduced?**
Yes

**Logs**
Crash log: https://pastebin.com/JmNqB6LT

**Issue**
Looking at a CC network cable with a Chisels & Bits Chisel in your hand causes the client to crash with the error:
IllegalStateException: The block computercraft:peripheral is multipart-compatible but its TileEntity isn't!

I presume holding bits will do it also.

**Comment by Frikkle** [comment]
Tried reproducing this on 1.63a using a few pieces of Networking Cable and bot a Diamond Chisel from Chisel and a Diamond Chisel from Chisel and Bits. Could not reproduce.

In my test setup the cabling is not connected to anything though.

**Comment by Kiefy** [comment]
I got a friend to try it and he crashed too.

This is the setup: 
![cc peripheral setup](https://i.imgur.com/LtPyYo7.png)

**Comment by Frikkle** [comment]
1.63 was changed to an alpha release and it has some issues; so I wouldn't recommend updating and waiting for 1.64 which hopefully is around the corner. I might look further into reproducing this issue tomorrow in SP mode.

**Comment by Frikkle** [comment]
I cannot seem to reproduce this issue. I've tried a few different setups but can't get it to crash the client.

@Kiefy Could you give me a set of reproducable steps? (so, 1. Place Advanced Computer, 2. Place wired modem (flat variant) on top of it. 3. etc. etc.) This way I can check the exact scenario and maybe help to pinpoint the cause.

@Kiefy Does the issue persist through server restarts?

@NillerMedDild Maybe you could try this on 1.62? I don't have an old instance set up at the moment.

**Comment by NillerMedDild** [comment]
Neither do I :/

**Comment by Kiefy** [comment]
I created a new SP world. placed an 8x2 adv monitor & 1x adv computer.  Connected the wired modem and r-clicked the monitor modem to connect. (monitor_0)
![Setup](https://i.imgur.com/tcmIEPe.png)

Then placed a Diamond Chisel on my hotbar. I was able to look at the modem on the computer. but as soon as I turned to the modem on the monitor, it crashed.

This is the screen at the point it froze.
![Freeze](https://i.imgur.com/g4W2aO4.png)

I did add some custom mods to the pack. I'll remove them, re-test & return back with my findings.
